packages/quantic/force-app/main/default/lwc/quanticResultAction/quanticResultAction.css

```diff
@@ -0,0 +1,40 @@
+.slds-nubbin_bottom-left:before,
+.slds-nubbin_bottom-left:after {
+  left: 50%;
+}
+
+.slds-popover_tooltip {
+  width: max-content;
+}
+
+.result-action_tooltip {
+  position: absolute;
+  top: -3rem;
```

Comment:
That is dependent on the font-size, have you tried changing font-size to see if that still remains aligned?

Reply:
good point! let me try that and see.